src/style/help/first-launch.less

```diff
@@ -364,6 +364,10 @@
     }
 }
 
+:lang(ja) .carousel__slide-button__continue {
+    bottom: 14rem;
+}
```

Comment:
This feels like not a robust fix for the problem. In your screenshot, the first line of text is wrapped to the second line because it is added a special header for translation purpose, and it is possible that the translated text will take only one line, and will break your fix again. Same thing could happen for FR and DE. Ideally, the "continue" button should flow with the text, instead of having a fixed position, which is the root of all the problem.